src/client_features.rs

```diff
@@ -631,6 +636,28 @@ mod tests {
     pub enum EdgeError {
         SomethingWentWrong,
     }
+    #[test]
+    pub fn can_deserialize_numbers_to_strings() {
+        let json = serde_json::json!({
+            "context": {
```

Comment:
Can we add some entries for the non properties fields too? They're handled distinctly enough that they cause problems and are worth a test

Reply:
good point, we might actually want to coerce the other non-properties fields to strings as well, while still supporting at least numbers? Though I guess when part of Query string they will always be strings, whereas when they're in the body, they can be either.

Reply:
This has now received the same treatment - booleans and numbers get converted to strings, empty strings, objects, and nulls get None